packages/astro/src/runtime/server/render/any.ts

```diff
@@ -18,6 +18,7 @@ export async function renderChild(destination: RenderDestination, child: any) {
 			});
 		});
 		for (const childRender of childRenders) {
+			if (!childRender) continue;
```

Comment:
We don't have to wait for @bluwy to weigh in, but it seems like some of the `childRender` objects are `undefined` which is kind of unexpected. Not sure I fully grok why that's happening? Is it Markdown-specific?